modules/KalturaSupport/components/pipBtn.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Created by itayk on 7/26/15.
+ */
+( function( mw, $ ) {"use strict";
+
+	mw.PluginManager.add( 'pipBtn', mw.KBaseComponent.extend({
+
+			defaultConfig: {
+				'parent': 'controlsContainer',
+				'accessibleControls': false,
+				'order': 5,
+				'showTooltip': true,
+				'align': "right",
+				"displayImportance": "medium"
+			},
+
+			isDisabled: true,
+			pipTitle: gM( 'mwe-embedplayer-pip' ),
+			setup: function() {
+				var _this = this;
+				this.bind("playerReady",function(){
+					_this.onDisable();
+				});
+				this.bind("firstPlay",function(){
+					 _this.onEnable();
+				});
+			},
+			isSafeEnviornment: function(){
+				return mw.isIOS9() && mw.isIpad() && !mw.isIpad2() ;
+			},
+			getComponent: function() {
+				var _this = this;
+				if( !this.$el ) {
+					this.$el  = $( '<button />' )
+						.attr( 'title', this.nextTitle )
+						.addClass( "btn btnNarrow icon-new-tab pull-right" )
```

Comment:
@ElizaSapir - please remove the btnNarrow class - it causes the button to look bad and is not needed here